react/components/QuickOrderAutocomplete.tsx

```diff
@@ -116,17 +114,17 @@ const QuickOrderAutocomplete: FunctionComponent<
     value: !term.length
       ? []
       : optionsResult
-          .filter((item: any) => {
-            return !!item.items[0].images[0].imageUrl
-          })
-          .map((item: any) => {
-            return {
-              value: item.items[0].itemId,
-              label: item.items[0].name,
-              slug: item.linkText,
-              thumb: getImageSrc(item.items[0].images[0].imageUrl),
-            }
-          }),
+        .filter((item: any) => {
+          return !!item.items[0].images[0].imageUrl
+        })
+        .map((item: any) => {
+          return {
+            value: item.productId,
```

Comment:
Might be better to just use the skuId, like it was before, to avoid any possibility of the same product returning twice and duplicating values.

No need to change the value, just the label.